implementations/micrometer-registry-statsd/src/main/java/io/micrometer/statsd/StatsdMeterRegistry.java

```diff
@@ -92,8 +93,13 @@ public class StatsdMeterRegistry extends MeterRegistry {
 
     FluxSink<String> sink = new NoopFluxSink();
 
+    Flux<String> flux = Flux.empty();
```

Comment:
It doesn't look like this is used. Am I missing something?

Reply:
Yes, you are right. I just kind of forgot it there from some previous attempt.